File: BikeShop.Frontend/webclient/src/entities/models/Acts/SupplyInvoice/SupplyInvoiceDTO.ts
import {SupplyInvoice} from "../../../entities/Acts/SupplyInvoice/SupplyInvoice"
import {SupplyInvoiceProduct} from "../../../entities/Acts/SupplyInvoice/SupplyInvoiceProduct"

export interface SupplyInvoiceDTO {
    supplyInvoice: SupplyInvoice
    supplyInvoiceProducts: SupplyInvoiceProduct[]
}

File: BikeShop.Backend/BikeShop.Acts/BikeShop.Acts.Domain/DTO/Requests/SupplyInvoice/Create/SupplyInvoiceDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BikeShop.Acts.Domain.DTO.Requests.SupplyInvoice.Create
{
    public class SupplyInvoiceDTO
    {
        public int ShopId { get; set; }
        public Guid User { get; set; }
        public decimal DeliveryPrice { get; set; }
        public decimal AdditionalPrice { get; set; }
        public string Description { get; set; } = string.Empty;
    }
}
